File: app/src/main/java/com/juniperphoton/myerlist/api/response/CheckUserResponse.java
package com.juniperphoton.myerlist.api.response;


import com.google.gson.annotations.SerializedName;

public class CheckUserResponse extends CommonResponse {
    @SerializedName("isExist")
    private boolean mIsExist;

    public CheckUserResponse() {

    }

    public boolean getExist() {
        return super.mIsSuccessed && mIsExist;
    }
}

File: app/src/main/java/com/juniperphoton/myerlist/api/response/CheckUserResponse.kt
package com.juniperphoton.myerlist.api.response

import com.google.gson.annotations.SerializedName

class CheckUserResponse : CommonResponse() {
    @SerializedName("isExist")
    private val isExist: Boolean = false

    val exist: Boolean
        get() = super.ok && isExist
}